modules/validations/empty_start_date.js

```diff
@@ -0,0 +1,49 @@
+import { t } from '../core/localizer';
+import { utilDisplayLabel } from '../util';
+import { validationIssue, validationIssueFix } from '../core/validation';
+
+export function validationEmptyStartDate() {
+  const type = 'empty_start_date';
+
+  const validation = function checkEmptyStartDate(entity) {
+    // If start_date is not empty, return nothing
+    if (entity.tags && (entity.tags.start_date || entity.tags['start_date:edtf'])) return [];
+    // If entity has no tags, return nothing
+    if (Object.keys(entity.tags).length === 0) return [];
```

Comment:
This is a good case to check for, but there are some edge cases it won’t cover. A node that’s part of a way, or [a way forming part of a boundary](https://www.openhistoricalmap.org/way/199746092) could have several keys such as `source` and `license`, but it doesn’t need a `start_date` because the parent elements would all have `start_date`s, and those dates could potentially differ. The validator rule about a missing primary feature tag, which serves a similar purpose as the rule you’re writing, uses this heuristic to exempt elements that are part of a larger whole:

https://github.com/OpenHistoricalMap/iD/blob/adb4d497c94d9fc85d60805ddfaaf46e9896b32f/modules/validations/missing_tag.js#L54-L57